File: Deliverables/BizBong/app/src/main/java/gamesoftitalia/bizbong/entity/Statistiche.java
package gamesoftitalia.bizbong.entity;

import java.util.List;

/**
 * Created by GameSoftItalia on 09/01/2017.
 */

public class Statistiche {

    private String[] modalitaList;
    private int[] punteggiList;

    public Statistiche(String[] modalitaList, int[] punteggiList){
        this.modalitaList = modalitaList;
        this.punteggiList = punteggiList;
    }

    public String[] getModalitaList() {
        return modalitaList;
    }

    public void setModalitaList(String[] modalitaList) {
        this.modalitaList = modalitaList;
    }

    public int[] getPunteggiList() {
        return punteggiList;
    }

    public void setPunteggiList(int[] punteggiList) {
        this.punteggiList = punteggiList;
    }
}

File: Deliverables/php/Entity/Statistiche.php
<?php
	//**********************************************************************************************
    // Creazione dell' oggetto Statistiche che implementa JsonSerializable per essere inviato tramite json
    // Variabili: modalitaList, punteggiList
    // jsonSerialize() --> utilizzato per la conversione a json
    //**********************************************************************************************
	
    //include ".../Libreria/ArrayList.php";
    
    class Statistiche implements JsonSerializable {
    	private $modalitaList;
    	private $punteggiList;
        
        public function Statistiche($punteggiList) {
        	$this->modalitaList = array("storia", "geografia", "sport", "scienza", "cinema e spettacolo",
            		"letteratura", "videogames", "cartoni e serie tv", "arte", "cucina", 
                    "classica", "bizbong");
            $this->punteggiList = $punteggiList;
        }
        
        public function getModalitaList() {
        	return $this->modalitaList;
    	}

        public function getPunteggiList() {
            return $this->punteggiList;
        }

        public function setPunteggiList($punteggiList) {
            $this->punteggiList = $punteggiList;
        }
              
        public function castObjectToStatistiche($obj){
            foreach($obj as $property => &$value){
            	$this->$property = &$value;
            }
        }
              
        public function jsonSerialize() {
        	return get_object_vars($this);
    	}
    }
?>